site/docs/components/combo-box/examples.mdx

```diff
@@ -176,5 +176,13 @@ When the `value` prop is an object, the `valueToString` prop must be provided to
 
 Use the `truncate` prop when you are limited on space and want to prevent the multi-select combobox from spanning multiple lines.
 
+</LivePreview>
+
+<LivePreview componentName="combo-box" exampleName="ClearAll" >
+
+## Clear All
+
+In order to achieve the clear all functionality, you can pass a close icon in endAdornment prop.
```

Comment:
This guidance is incorrect, a button is passed to endAdornment not an icon. Also we typically use backticks (`) when referencing props.